elixir/apps/domain/test/domain/auth_test.exs

```diff
@@ -1806,6 +1806,65 @@ defmodule Domain.AuthTest do
       assert Repo.aggregate(Auth.Identity, :count) == 0
       assert Repo.aggregate(Domain.Actors.Actor, :count) == 0
     end
+
+    test "resolves provider identifier conflicts across actors", %{
```

Comment:
@jamilbk this is because of the bug that we had before d406f603f4c3d727343d936e6b8d0bf6066ae3fd, I also manually fixed conflicts in the database but decided to still write a hotfix in case something will go wrong in the future